packages/web3-eth-personal/src/rpc_method_wrappers.ts

```diff
@@ -73,7 +73,9 @@ export const sendTransaction = async (
 	tx: Transaction,
 	passphrase: string,
 ) => {
-	const formattedTx = formatTransaction(tx, ETH_DATA_FORMAT);
+	const formattedTx = formatTransaction(tx, ETH_DATA_FORMAT, {
```

Comment:
I suggested in above comments in request manager to lets remove config from request manager so with that , this file also needs change, and in `Personal` class use context, to get tx formatting schema from context and pass to these functions, and it should be optional so its not breaking change.